tailwind.config.ts

```diff
@@ -2,8 +2,8 @@ import typography from "@tailwindcss/typography";
 import type { Config } from "tailwindcss";
 
 export default {
-  content: ["./src/**/*.{js,jsx,ts,tsx}"],
-  darkMode: "class",
+  content: ["./src/**/*.{js,jsx,ts,tsx,md,mdx}", "./docs/**/*.{md,mdx}"],
```

Comment:
This change allows us to use css classes in MDX files without adding them to an exception or other `js` file. Any classes not currently used in js, jsx, ts, or tsx files will be removed from tailwind styles. Now the preprocessor looks for classes in md and mdx files and includes those in the final build